### PR TITLE
feat: config option to expand/collapse the examples payload

### PR DESCRIPTION
**Pull request by AceTheCreator**
**Description**

Changes proposed in this pull request:
Ability to configure the default expand/collapse behavior for the "Examples" section.

**Related issue(s)**
#532

### CODE REVIEW

**Comment by AceTheCreator** [comment]
@magicmatatjahu kindly review this PR. Let me know what you think :)

File: library/src/config/config.ts [hunk 1]
```diff
@@ -15,6 +16,10 @@ export interface ShowConfig {
   errors?: boolean;
 }
 
+export interface ExpandedConfig {
```
Comment (magicmatatjahu) [inline library/src/config/config.ts:19]:
```suggestion
export interface ExpandConfig {
```

File: library/src/containers/Messages/Message.tsx [hunk 2]
```diff
@@ -32,7 +32,6 @@ export const Message: React.FunctionComponent<Props> = ({
   showExamples = false,
 }) => {
   const config = useConfig();
-
```
Comment (magicmatatjahu) [inline library/src/containers/Messages/Message.tsx:35]: Leave that empty line :)

File: library/src/containers/Messages/MessageExample.tsx [hunk 3]
```diff
@@ -49,7 +50,10 @@ export const Example: React.FunctionComponent<ExampleProps> = ({
   schema,
   examples = [],
 }) => {
-  const [expand, setExpand] = useState(false);
+  const config = useConfig();
+  const [expand, setExpand] = useState(
+    config && config.expand && config.expand.messageExamples,
```
Comment (magicmatatjahu) [inline library/src/containers/Messages/MessageExample.tsx:55]:
```suggestion
    (config && config.expand && config.expand.messageExamples) || false,
```

File: playground/src/common/defaultConfig.ts [hunk 4]
```diff
@@ -7,5 +7,8 @@ export const defaultConfig: string = `{
     "messages": true,
     "schemas": true,
     "errors": true
+  },
+  "expanded":{
+    "messageExamples": false,
   }
```
Comment (magicmatatjahu) [inline playground/src/common/defaultConfig.ts:13]: Here you should have valid JSON:

```suggestion
  "expand": {
    "messageExamples": false
  }
```

and probably by this config for playground didn't work for you.

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]
Awesome 🚀  but we need improve some things :)

**Comment by magicmatatjahu** [comment]
@AceTheCreator Please also update documentation https://github.com/asyncapi/asyncapi-react/blob/next/docs/configuration/config-modification.md

**Comment by AceTheCreator** [comment]
> @AceTheCreator Please also update documentation https://github.com/asyncapi/asyncapi-react/blob/next/docs/configuration/config-modification.md

@magicmatatjahu is the PR fine :)

**Pull request merged**